arm9/overlays/24/include/overlay_24.h

```diff
@@ -0,0 +1,81 @@
+#ifndef POKEDIAMOND_OVERLAY_24_H
+#define POKEDIAMOND_OVERLAY_24_H
+
+typedef enum {
+    STYLUS_TYPE_ERASE = 0,
+    STYLUS_TYPE_DRAW = 1,
+} StylusType;
+
+typedef enum {
+    MEMO_PAD_PIXEL_TYPE_FILLED = 1,
+    MEMO_PAD_PIXEL_TYPE_EMPTY = 4,
+} MemoPadPixelType;
+
+typedef struct MemoPadDrawState MemoPadDrawState;
```

Comment:
merge the typedefs and the structs below, unless there's a circular dependency for the struct